External logging configured in CM (#4252)
strimzi/strimzi-kafka-operator: documentation/modules/appendix_crds.adoc

It must have the value `external` for the type `ExternalLogging`.
[options="header"]
|====
|Property          |Description
|type       1.2+<.<|Must be `external`.
|string

> |name       1.2+<.<|*The property `name` has been deprecated. This feature should now be configured at path `logging.valueFrom.configMapKeyRef.name`.* The name of the `ConfigMap` from which to get the logging configuration.

Reviewer: ```suggestion
|name       1.2+<.<|*The `name` property has been deprecated, and should now be configured using `logging.valueFrom.configMapKeyRef.name`.* The name is the `ConfigMap` from which to get the logging configuration.
```
